networkx/utils/heaps.py

```diff
@@ -64,7 +64,7 @@ def pop(self):
         """
         raise NotImplementedError
 
-    def get(self, key, default=None):
+    def get(self, key, default=1):
```

Comment:
Hmm, why this change? What is this pull request supposed to do?

Reply:
My bad. I was getting frustrated because the test kept failing and I didn't link the issue. It's #6443. Have I implemented it wrongly?

Reply:
Yes, this bit of code is for heaps. I don't think this has anything to do with GEXF export.

What @dschult meant in https://github.com/networkx/networkx/pull/6484#issuecomment-1465016028 was to look at dict.get in python https://docs.python.org/3/library/stdtypes.html#dict.get

An example:
``` python
>>> d = {1: 1, 2: 2}
>>> d.get(1)
1
>>> d.get(3)  # by default it gives None
>>> d.get(3, 'something') # you can change the default by providing another value
'something'
```

Reply:
smh. My bad. Thank you for explaining. will correct it now.